services/bots.py

```diff
@@ -117,10 +117,32 @@ def get_token_type_mapping(repo: Repository):
             "Repository has no good bot for admin, but still continuing operations in case it is not doing an admin call anyway",
             extra=dict(repoid=repo.repoid),
         )
+
+    if (
+        repo.service == "github"
+        and get_config("github", "integration", "id")
+        and get_config(
+            "github",
+            "integration",
+            "default_installation_id",  # this is different from the integration id,
```

Comment:
is this something new that you are adding or a piece of config that already exists?